src/Stratis.Bitcoin.FullNode.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29020.237
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.539
```

Comment:
What is this change?

Reply:
MSVS did that itself. Can revert if it will be an issue.